oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/type/event/EventQueryCondition.java

```diff
@@ -43,5 +43,7 @@ public class EventQueryCondition {
 
     private Order order;
 
+    private String layer;
```

Comment:
I think you should hard code Layer.UNDEFINED in here, which means minimal changes in irrelevant things, and help @fgksgf to avoid conflicts as much as possible. WDYT?

Reply:
Sure, I have updated.